bindings/nodejs/lib/wallet/wallet.ts

```diff
@@ -148,6 +148,17 @@ export class Wallet {
         return this.methodHandler.getClient();
     }
 
+    /**
+     * Get chrysalis data.
+     */
+    async getChrysalisData(): Promise<Record<string, string>> {
```

Comment:
Is Record a hashmap?

Reply:
Not necessarily I think, but it's a type that can be used for this https://stackoverflow.com/a/62650503